Library/ViewModels/LoginToutViewModel.swift

```diff
@@ -97,6 +97,10 @@ public protocol LoginToutViewModelOutputs {
 
   /// Emits an access token to show 2fa view when Facebook login fails with tfaRequired error
   var startTwoFactorChallenge: Signal<String, Never> { get }
+
+  /// True if the feature flag for OAuth login is true.
+  /// Note that this is not a signal, because we don't want it to ever change after the screen is loaded.
+  var loginWithOAuthEnabled: Bool { get }
```

Comment:
This is perhaps a bit of a paranoid move, but since login is something you might do _right after_ launch, I didn't want there to ever be a situation where the remote config reloads after the login screen has already been presented.